src/pages/MsigUI.vue

```diff
@@ -88,22 +154,95 @@ export default defineComponent({
 			...toRefs(state),
 			key,
 			onTabChange,
+			mparamsPartial,
+			mparamsSigned,
 		};
 	},
 	methods: {
-		sign() {
+		async sign() {
+			let TxnBase64: string;
+			TxnBase64 = "";
+			let signedTxn: JsonPayload;
+			TxnBase64 = this.unsignedJson;
```

Comment:
I see we only need one TxnBase64 assign statement, right?

Reply:
yes, that's right